game/scripts/npc/abilities/ogre_magi_ignite.txt

```diff
@@ -37,17 +37,17 @@
       "01"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "duration"                                        "5 6 7 8 10 15"
+        "duration"                                        "5 6 7 8 9 10"
       }
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "burn_damage"                                     "26 34 42 50 90 130"
+        "burn_damage"                                     "26 34 42 50 150 250"
```

Comment:
This is a buff actually damage-wise. Also you should not change this because of the new mechanic: "Ignite hitting an already affected unit now adds to the duration, rather than just refreshing it". In other words its infinite aids unless you dispel it.

Reply:
Reduced buff. Same damage level 5 and more level 6. (Pipe + Luciences will outheal the damage). Isn't it good that im nerfing the spell duration and slow due to the ignite change?.

Reply:
If I don't comment on that then its good :D He definitely needed that. I remember when it was 20 seconds or something at max level.

Reply:
yikes